README.md

```diff
@@ -1,2 +1,337 @@
 # fearless-utils-Android
 fearless-utils-Android (This repository is managed by Terraform!)
```

Comment:
Fearless Wallet utils Android is a native Android library to help developers built native mobile apps for Substrate-based networks, e.g. Polkadot, Kusama, and Westend.